src/storage/FileIdentifierMapper.ts

```diff
@@ -1,21 +1,40 @@
 import type { ResourceIdentifier } from '../ldp/representation/ResourceIdentifier';
 
+export interface ResourceLink {
+  /**
+   * Identifier of a resource.
+   */
+  identifier: ResourceIdentifier;
+  /**
+   * File path of a resource.
+   */
+  filePath: string;
+  /**
+   * Content-type for a data resource (not defined for containers).
+   */
+  contentType?: string;
```

Comment:
Shall we make up our own internal content type for containers for simplicity?

Reply:
Not sure since this is the only place this would be used. A Representation corresponding to a container is going to have a specific existing content-type since there the metadata gets returned in the body, so this might cause some confusion then.

Reply:
Agreed.